ccloud-observability/docs/general-scenarios.rst

```diff
@@ -8,4 +8,5 @@ General Client Scenarios
    :titlesonly:
 
    Overview <general-scenarios/overview>
-   Failing to create a new partition <general-scenarios/fail-to-create-partition>
\ No newline at end of file
+   Failing to create a new partition <general-scenarios/fail-to-create-partition>
+   Request rate limits <general-scenarios/request-rate-limits>
```

Comment:
With this new page, were you thinking to also trim down docs/producer-scenarios/confluent-cloud-unreachable.rst to remove the throttle-related metrics from that page?

Reply:
Addressed in https://github.com/confluentinc/examples/pull/952/commits/67d43da96fa0a7d3457747222007490f039bc799